assets/config.php

```diff
@@ -118,6 +118,16 @@
             'public' => '%storage_path%/app/public/',
         ],
 
+        /*
+         * This makes local driver disks support tenant-aware Storage::disk()->url().
+         * Use `php artisan tenants:link` to create a symbolic link from the tenant's storage to his public directory.
+         */
+        'url_override' => [
+            // 'local disk' => 'public directory' (%tenant_id% will be replaced with actual tenant id)
+            // The local disk must exist in the tenancy.filesystem.root_override config
+            'public' => 'public-%tenant_id%',
+        ],
```

Comment:
I think this section should be a bit better explained. For the other keys like this we have some examples I believe

Reply:
Updated the comments for this config key https://github.com/archtechx/tenancy/pull/909/commits/f2d562cd8b81255877d14ecb92f351e9c78d8b4f